lib/board.rb

```diff
@@ -0,0 +1,49 @@
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		@cells = Array.new(9, " ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.